### PR TITLE
Follow button

### PR DESCRIPTION
**Pull request by sumeetadur**
Allows the option for a follow button. Created when a “followButton” object is used in the section object, e.g.:

```javascript
followButton: {
	conceptId: 'NDdiMzAyNzctMTRlMy00Zjk1LWEyZjYtYmYwZWIwYWU2NzAy-VG9waWNz',
	name: 'UK\'S EU referendum',
	taxonomy: 'topics',
	classes: 'brexit'
}
```

This will be used in the upcoming Brexit home page component (currently being developed [here](https://github.com/ft-interactive/brexit-front-page))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "license": "MIT",
   "dependencies": {
     "@financial-times/n-card": "^3.0.0",
+    "next-myft-ui": "^6.20.1",
```
Comment (laurieboyes) [inline package.json]: @wheresrhys would we rather this dependency was via n-ui (new Next UI monorepo)?

Comment (wheresrhys) [inline package.json]: mmm sort of yes, but as it's still not quite at v1 leave it for now

File: components/section/meta/meta.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+import Follow from 'next-myft-ui/templates/follow';
 
 export default class extends Component {
 	render () {
 		return (
 			<div className="section-meta">
 				<h2 className="section-meta__title" dangerouslySetInnerHTML={{ __html: this.props.title }} />
+				{
+					this.props.followButton ?
+						<Follow conceptId={this.props.followButton.conceptId} name={this.props.followButton.name} taxonomy={this.props.followButton.taxonomy} classes={this.props.followButton.classes} /> :
```
Comment (i-like-robots) [inline components/section/meta/meta.js]: `<Follow {...this.props.followButton} />`

File: components/section/meta/meta.js [hunk 3]
```diff
@@ -1,10 +1,16 @@
 import React, { Component } from 'react';
+import Follow from 'next-myft-ui/templates/follow';
 
 export default class extends Component {
```
Comment (i-like-robots) [inline components/section/meta/meta.js]: Maybe make this a stateless component while you're here?

**Comment by i-like-robots** [comment]
LGTM - would squash before merging =]

**Pull request merged**